Lib/test/test_uuid.py

```diff
@@ -4,12 +4,17 @@
 import contextlib
 import io
 import os
+import pickle
 import shutil
 import subprocess
+import sys
 
 py_uuid = support.import_fresh_module('uuid', blocked=['_uuid'])
 c_uuid = support.import_fresh_module('uuid', fresh=['_uuid'])
-
+# would prefer using platform.system() but CI tests have a merge conflict with "import platform"
```

Comment:
Merge/rebase this PR branch to overcome merge conflicts, and use the most appropriate code here.

Reply:
Why do you prefer `platform.system()` over `sys.platform`? What value does `sys.platform` have on AIX? If it is reasonable ISTM that we should use `sys.platform` here.